bin/all_sky_search/pycbc_template_kde_calc

```diff
@@ -161,17 +163,39 @@ with h5py.File(args.template_file, "r") as f_src:
 temp_samples = numpy.vstack((template_pars)).T
 
 
+# Rejection Sampling
+if 'mchirp' not in args.fit_param:
+    raise ValueError("mchirp does not exist in args.fit_param")
```

Comment:
But it's OK to not have mchirp as a parameter _if we are not doing the rejection sampling_. So these lines and the assignment of mchirp_index should be inside the block conditional on `mchirp_downsample_power is not None`.